modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/porosity_jump/formloss/formloss_multiple.i

```diff
@@ -0,0 +1,121 @@
+# The expected pressure drop from textbook correlations for this example is: 1081.468315
```

Comment:
Can you add the expression?
Should be K times some integral across the 2D channel?

Reply:
Added!

Reply:
This is a 1D channel in diguise. It is the normal velocity.